src/Network/Network/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 --->
 
 ## Upcoming Release
+* Added public ip address as a mandatory parameter to create route server
```

Comment:
Please update change log as PublicIpAddress is optional now.

Reply:
updated